ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -663,6 +663,12 @@
   private static final String KSQL_ENDPOINT_MIGRATE_QUERY_DOC
       = "Migrates the /query endpoint to use the same handler as /query-stream.";
 
+  public static final String KSQL_COMMAND_TOPIC_RATE_LIMIT_CONFIG
+      = "ksql.command.topic.rate.limit";
+  public static final double KSQL_COMMAND_TOPIC_RATE_LIMIT_CONFIG_DEFAULT = 10.0;
```

Comment:
Nit: Do we need a double or can we use an int?

Reply:
the library takes a double as `permitsPerSecond` https://guava.dev/releases/19.0/api/docs/index.html?com/google/common/util/concurrent/RateLimiter.html

Reply:
Can we set the default much higher (like 1000)? I think 10 is too low for a default and many users may inadvertently hit this.

Reply:
Also, could you move the config to KsqlRestConfig and follow a similar naming pattern to `KSQL_COMMAND_RUNNER_BLOCKED_THRESHHOLD_ERROR_MS` and `DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_CONFIG` with `ksql.server.command` prefix? I think we should keep the command topic/command runner configs together and named consistently

Reply:
sure, moved it. I upped the limit for now, @rodesai WDYT about doing double max value versus 10? seems ok to me

Reply:
Yes we should definitely disable rate-limiting by default by setting this to MAX_INT. We can enable in cloud as needed.